src/style.css

```
@@ -0,0 +1,112 @@
+/* #region normalize*/
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+body,
+p {
+  margin: 0;
+}
+/* #endregion*/
+
+/* #region card*/
+.card {
+  width: 200px;
+  padding: 14px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 16px 5px 45px ;
+}
+
+.card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  margin-bottom: 5px;
+  line-height: 15px;
+}
+
+.card__product-code {
+  margin-bottom: 15px;
+}
+/* #endregion*/
+
+/* #region rating*/
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.rating-stars {
+  display: flex;
+}
+
+.rating-stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-size: none;
+  background-position: center;
+}
+
+.rating-stars__star--active {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+/* #region price*/
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.price__value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+/* #endregion */
+
+/* #region button*/
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
```

```suggestion
.card__button {
  display: flex;
  justify-content: center;
  align-items: center;
```
выравнивание уже сделано флексами - писать еще одно свойство нет смысла

---

agree